regexes/device/mobiles.yml

```diff
@@ -2203,19 +2233,19 @@ ECS:
 
 # Elephone
 Elephone:
-  regex: 'Elephone[ _\-]([^/;]+) Build|P6000( ?Pro| ?Plus|\+| ?02)? Build'
+  regex: 'Elephone|P6000( ?Pro| ?Plus|\+| ?02)? Build'
   device: 'smartphone'
   models:
     - regex: 'Elephone[ _\-]([^/;]+) Build'
       model: '$1'
-    - regex: 'P6000 Build'
-      model: 'P6000'
     - regex: 'P6000(\+| ?Plus) Build'
       model: 'P6000 Plus'
     - regex: 'P6000 02 Build'
       model: 'P6000 02'
     - regex: 'P6000 ?Pro Build'
       model: 'P6000 Pro'
+    - regex: '(P[68]000)'
```

Comment:
`P8000` is currently only detected if the useragent also contains `Elephone`. Is that on purpose? Otherwise it should be added the the brand regex as well..

Reply:
Yes, it was on purpose.